geode-core/src/integrationTest/java/org/apache/geode/internal/offheap/OffHeapRegionBase.java

```diff
@@ -475,7 +475,8 @@ private void doRegionTest(final RegionShortcut rs, final String rName, boolean c
           fail("identity of offHeapStore changed when cache was recreated");
         }
         r = gfc.createRegionFactory(rs).setOffHeap(true).create(rName);
-        assertTrue(ma.getUsedMemory() > 0);
+        await()
+            .untilAsserted(() -> assertTrue(ma.getUsedMemory() > 0));
```

Comment:
Please use AssertJ so it prints out a more detailed failure message:
```
await().untilAsserted(() -> {
  assertThat(ma.getUsedMemory()).isGreaterThan(0);
});
```

Reply:
Thank you for the review. I have applied all of your comments.